src/Symfony/Component/Yaml/Tests/DumperTest.php

```diff
@@ -580,6 +595,43 @@ public function testNoExtraTrailingNewlineWhenDumpingAsMultiLineLiteralBlock()
         $this->assertSame($data, Yaml::parse($yaml));
     }
 
+    public function testDumpTrailingNewlineInMultiLineLiteralBlocksForTaggedValues()
+    {
+        $data = [
+            'clip 1' => new TaggedValue('my-tag', "one\ntwo\n"),
+            'clip 2' => new TaggedValue('my-tag', "one\ntwo\n"),
+            'keep 1' => new TaggedValue('my-tag', "one\ntwo\n"),
```

Comment:
aren't these 3 values the same ?

Reply:
@stof correct, this is a copy/paste from the test `testDumpTrailingNewlineInMultiLineLiteralBlocks()` below.

@xabbuh do you remember why these values are identical? Should update both tests?

Reply:
As far as I know there was no special reason. I simply forgot to clean up the test before committing. Feel free to shorten it.

Reply:
@stof @xabbuh sorry for the delay, I've pushed changes to remove the duplicates in the tests.